### DISCUSSION TITLE
Code colors

### DISCUSSION THREAD

**Discussion by joncode21** [phcode-dev/phoenix#1810] [General]
Hi 

Can there be a manual or a link I can refer  to for the kind of colors  that are in the code editor?

**Comment by abose** [comment]
@joncode21 When you mean colors, did you mean the colors that you get for code highlighting or something else?

**Reply by joncode21** [reply]
Like when I type codes, there are different colours that appear.Can u create a doc like blue represents what, pink represents what?
